github-issue-summary-to-email/main.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/http;
```

Comment:
Code is not properly formatted, use ballerina formatter.

Reply:
Updated in [5131705](https://github.com/ballerina-guides/integration-samples/pull/35/commits/51317055927a041c645786b20bcb6af3864f1081). For future references, what is the recommended method for formatting? After the review I used the the command `bal format` but previously I executed `format document` option in VS code in this code.